### ISSUE TITLE
koboldcpp does not use a gpu

### ISSUE THREAD

**Issue by RaSH3060** [LostRuins/koboldcpp#366]
koboldcpp does not use the video card, because of this it generates for a very long time to the impossible, the rtx 3060 video card. When choosing Presets: Use CuBlas or CLBLAS crashes with an error, works only with NoAVX2 Mode (Old CPU) and FailsafeMode (Old CPU) but in these modes no RTX 3060 graphics card enabled
CPU Intel Xeon E5 1650
RAM 32Gb
GPU RTX 3060 12GB
![Screenshot_22](https://github.com/LostRuins/koboldcpp/assets/141317474/880a351d-bda5-46ab-b637-b22f149521c3)
![Screenshot_23](https://github.com/LostRuins/koboldcpp/assets/141317474/5336ae84-a9f0-4085-97e9-34533490cda5)

**Comment by LostRuins** [comment]
Yeah, currently GPU acceleration is not enabled for noavx2 compatibility mode. Your CPU is too old to use the AVX2 instruction. Eventually I may add that as a supported combination, but currently it is not possible.

**Comment by Animus777** [comment]
I'm in the same boat with Sandy Bridge i5 2500k. I wonder what would be more likely solution: Vulkan support or Cubals compatible with avx1?

**Comment by LostRuins** [comment]
Finally added in v1.48, please try

**Comment by TFWol** [comment]
> Finally added in v1.48, please try

I did a quick test and it works for me.
People should keep in mind to check `Disable MMAP` if they want RAM offload to VRAM to work.
